logging/advancedlog/main.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START logging_write_log_entry_advanced]
+
+// Writes an advanced log entry to Cloud Logging.
+package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+
+	"cloud.google.com/go/logging"
+)
+
+func main() {
```

Comment:
Add a test? See https://github.com/GoogleCloudPlatform/golang-samples/blob/master/internal/testutil/runmain.go (not required to use it, but it may be helpful).

Reply:
Oh neat. Added.

I test that the main executed without error or stderr logs. I did not test get-logs again. Figured it's redundant since I updated simplelog to run the same test. Also, it would take 15 seconds run.